specs/0108-community-health-dashboard.md

```diff
@@ -33,19 +33,29 @@ Active users within a selected time frame are defined like this:
 
 Note that only users included in the all users counts can be active users. If a project contact person rejects a suggestion or submits a translation, it doesn't count as an active reviewer or contributor of the team.
 
-## Unreviewed suggestion lifespan
+## Average age of currently unreviewed suggestions
 
-A line chart showing the average age of the unreviewed suggestions in a particular month for a period of last 12 months.
+A line chart showing the average age of suggestions still unreviewed in a particular month, over the last 12 months.
 
-Data is available for each day, but aggregated by month in the chart.
+Data is available for each day, but aggregated by month in the chart. A tooltip showing the exact data at a given month appears when hovering over a chart.
+
+**Text for infobox**: How old are on average the suggestions currently unreviewed.
+
+## Time to review suggestions
+
+A line chart displaying two separate data points over the last 12 months:
+* *Age of reviewed suggestions*: average age of suggestions reviewed during the specific month.
+* *12-month average*: average age of suggestions reviewed in the 12 months before the specific month.
+
+In both cases, data is determined by looking at translations rejected or approved, ignoring self-approved or self-rejected translations.
 
-A tooltip showing the exact data at a given month appears when hovering over a chart.
+**Text for infobox**: The average number of days a contributor can expect to wait for their translation suggestions to be reviewed by managers and translators.
 
 ## Translation activity
 
 A combination of two charts:
 1. A line chart showing overall completion of team projects over time.
-1. A column chart showing translation submissions over time in a stack of two columns - human translations and machinery translations (submitted as unchanged copies of suggestions from Machinery). Translation submissions also include imported translation by sync. New source string additions are plotted in a separate column, which is hidden by default.
+2. A column chart showing translation submissions over time in a stack of two columns - human translations and machinery translations (submitted as unchanged copies of suggestions from Machinery). Translation submissions also include imported translation by sync. New source string additions are plotted in a separate column, which is hidden by default.
```

Comment:
This shouldn't be [necessary](https://www.markdownguide.org/basic-syntax/#ordered-lists). (Also below.)

Reply:
Can we keep it though? It's confusing as heck to have `1.`, which then magically turns into `2.` when previewing.

Reply:
You'll get used to it. 😆 OTOH, reordering items is faster if you always use `1.`. (Technically speaking, that's not true - [because even a random order is allowed](https://www.markdownguide.org/basic-syntax/#ordered-lists) - but then it gets _really_ confusing).

As with any code style, I don't have a strong preference. But there are two reasons why I don't think we should be making this change:

1. It's outside the scope of the PR.
2. For vast majority of code we have formatters in place, so we _don't need to pay any more attention to style during code reviews, and are free from those never-ending code style discussions (A. G.)_. I'm all for setting up a formatter for Markdown, but until we have one, I don't think imposing code style rules will make us more productive.

Reply:
![Powerful-Crying-Gif-From-Doctor-Who](https://user-images.githubusercontent.com/3644868/107246367-7a6efa80-6a30-11eb-8780-d3904f2342e9.gif)

Reply:
![](https://media.giphy.com/media/cIjssqJCX5Asbqy2H0/giphy.gif)